src/components/attendee/AttendeeLayout.tsx

```diff
@@ -36,6 +38,15 @@ interface _AttendeeLayoutProps {
 
 const _AttendeeLayout: React.FC<_AttendeeLayoutProps> = ({ space }) => {
   const [backButtonSpace, setBackButtonSpace] = useState<SpaceWithId>();
+  const footerRef = useRef<HTMLElement>(null);
+  const footerIntersect = useIntersection(footerRef, {
+    rootMargin: "0px",
+  });
+
+  const layerUIClasses = classNames(styles.LayerUi, {
+    [`${styles.LayerUi} ${styles.LayerUiAbsolute}`]: footerIntersect?.isIntersecting,
+  });
```

Comment:
For readability, helped me discover a duplicate:

```suggestion
  const layerUIClasses = classNames(styles.LayerUi, {
    [styles.LayerUi]: footerIntersect?.isIntersecting, 
    [styles.LayerUiAbsolute]: footerIntersect?.isIntersecting,
  });
```

And I think `styles.LayerUi` should be removed from one of these two places, it should either be always present or always dependent on the intersecting value

Reply:
done [1ec7d1e](https://github.com/sparkletown/sparkle/pull/2973/commits/1ec7d1e6854fb0c34f6633c95e8d4ec95fe9603c)